pytplot/tplot_math/resample.py

```diff
@@ -29,7 +29,7 @@
     """
 
                               
-def tplot_resample(tvar1,times,newtvar):
+def resample(tvar1,times,newtvar):
```

Comment:
Yep

Reply:
Wait actually, similar to my comment on the interpolate function, if the spec_bins are time varying, then resample those as well, and store them back in with store_data